### DISCUSSION TITLE
Additional Report Options / Categories

### DISCUSSION THREAD

**Discussion by 67372a** [civitai/civitai#166] [Ideas]
While we can block / hide users, it would be good to be able to specifically report users for some additional reasons such as spam, scams, phishing, harassment, soliciting, and trolling/inciting conflict. Other is always an option, but might be good to have additional specific options, for harassment and trolling in particular.

**Comment by maxhulker** [comment]
Hey, appreciate the feedback. 

I  can see the use case for this but if we add it, it'll have to be later on. a lot on the development plate right now. 

Do you feel like the "other" field suffices for now?

**Reply by 67372a** [reply]
Hey @maxhulker , appreciate the response, so long as the other field gets proper attention and review, it will suffice for now
